packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -61,6 +61,18 @@ async function handleDeleteContext(contextName: string) {
     }
   }
 }
+
+async function startMonitoring(contextName: string, contextNumber: number): Promise<void> {
+  await window.telemetryTrack('kubernetes.monitoring.start.non-current', {
+    contextNumber,
+  });
```

Comment:
question: not sure the goal of providing the context number ?

Reply:
The goal is to indicate if the user repeats the operation on a specific context, without giving its real name. But the index won't be fixed when deleting contexts, perhaps a hash would be better

Reply:
but I mean, does this information matters

like maybe it's more interesting to know the number of existing contexts as well I don't know. I mean, I'm not sure what we could extract knowing the index of a context

Reply:
I'm removing the index for now, and keeping only the `kubernetes.monitoring.start.non-current` code, as the main purpose is to know if users are using this button. If they do, we will be able to add more information